### PR TITLE
Added await to the IngestFromStorageAsync calls

### PR DESCRIPTION
**Pull request by grik001**
Added await to the IngestFromStorageAsync calls

### CODE REVIEW

**Comment by PRMerger17** [comment]
@grik001 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by YaelSchuster** [comment]
Hi @grik001, you're rocking this docs contribution! Thanks!
@kerend please review

File: data-explorer/ingest-json-formats.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Use C# to ingest data in raw [JSON format](#the-json-format).
             }
         };
 
-    ingestClient.IngestFromStorageAsync(blobPath, properties);
+    await ingestClient.IngestFromStorageAsync(blobPath, properties);
```
Comment (vladikbr) [inline data-explorer/ingest-json-formats.md:157]:
```suggestion
    await ingestClient.IngestFromStorageAsync(blobPath, properties).ConfigureAwait(false);
```

File: data-explorer/ingest-json-formats.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ In this example, you ingest JSON records data. Each JSON property is mapped to a
...
```
Comment (vladikbr) [inline data-explorer/ingest-json-formats.md:294]:
```suggestion
    await ingestClient.IngestFromStorageAsync(blobPath, properties).ConfigureAwait(false);
```

File: data-explorer/ingest-json-formats.md [hunk 3]
```diff
@@ -359,7 +359,7 @@ var properties =
         }
     };
 
-ingestClient.IngestFromStorageAsync(blobPath, properties);
+await ingestClient.IngestFromStorageAsync(blobPath, properties);
```
Comment (vladikbr) [inline data-explorer/ingest-json-formats.md:362]:
```suggestion
await ingestClient.IngestFromStorageAsync(blobPath, properties).ConfigureAwait(false);
```

File: data-explorer/ingest-json-formats.md [hunk 4]
```diff
@@ -496,7 +496,7 @@ Array data types are an ordered collection of values. Ingestion of a JSON array
             }
         };
 
-    ingestClient.IngestFromStorageAsync(blobPath, properties);
+    await ingestClient.IngestFromStorageAsync(blobPath, properties);
```
Comment (vladikbr) [inline data-explorer/ingest-json-formats.md:499]:
```suggestion
    await ingestClient.IngestFromStorageAsync(blobPath, properties).ConfigureAwait(false);
```

File: data-explorer/ingest-json-formats.md [hunk 5]
```diff
@@ -644,7 +644,7 @@ Dictionary structured JSON contains key-value pairs. Json records undergo ingest
                 IngestionMappingReference = tableMapping
             }
         };
-    ingestClient.IngestFromStorageAsync(blobPath, properties);
+    await ingestClient.IngestFromStorageAsync(blobPath, properties);
```
Comment (vladikbr) [inline data-explorer/ingest-json-formats.md:647]:
```suggestion
    await ingestClient.IngestFromStorageAsync(blobPath, properties).ConfigureAwait(false);
```

**Review by vladikbr — CHANGES_REQUESTED** [review]
See change suggestions

**Comment by PRMerger13** [comment]
@grik001 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by YaelSchuster** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@YaelSchuster: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**